src/tests/ftest/util/dmg_utils.py

```diff
@@ -36,6 +36,19 @@ class DmgCommand(DmgCommandBase):
     # pylint: disable=too-many-ancestors,too-many-public-methods
     """Defines a object representing a dmg command with helper methods."""
 
+    # Member state defined in control/system/member.go. Used for dmg system
+    # query.
+    UNKNOWN = 0
+    AWAIT_FORMAT = 1
+    STARTING = 2
+    READY = 3
+    JOINED = 4
+    STOPPING = 5
+    STOPPED = 6
+    EVICTED = 7
+    ERRORED = 8
+    UNRESPONSIVE = 9
```

Comment:
Are these are specific to system query?  Should they be assigned under a unique variable, e.g.
```
SYSTEM_QUERY_STATES = {
    "UNKNOWN": 0
    "AWAIT_FORMAT": 1
    "STARTING": 2
    "READY": 3
    "JOINED": 4
    "STOPPING": 5
    "STOPPED": 6
    "EVICTED": 7
    "ERRORED": 8
    "UNRESPONSIVE": 9
}
```

Reply:
Done